lib/routes/account.js

```diff
@@ -731,6 +731,10 @@ module.exports = function (
               redirectTo: redirectTo,
               resume: resume,
               acceptLanguage: request.app.acceptLanguage
+            }).then(() => {
+              return request.emitMetricsEvent('email.verification.sent', {
+                uid: emailRecord.uid.toString('hex')
+              })
```

Comment:
Seeing this has made me realise I'm guilty of bad API design.

That second argument is only used in the activity event path of execution, so we could save the effort of the `toString` call here. But that is a totally leaky abstraction, and is really a difference that I should have contained within the events module itself.

Anyway, in the meantime, feel free to ditch the second argument for these if you like.